guides/getting-started.md

```diff
@@ -92,14 +92,10 @@ In `angular.json`, place references for each font in your project's build target
 ```
```

Comment:
A note on a couple previous section while we're in here:

--------

Step 2B's code example should look like this:

```
@NgModule({
     imports: [CashmereModule],
     exports: [CommonModule, CashmereModule]
})
export class SharedModule {}
```


-----
Step 3C could be changed to be something like below. Important tweak to node_modules path- should be single "./" rather than "../" since node_modules is usually a sibling to angular.json

```
"styles": [
                "./node_modules/font-awesome/css/font-awesome.css",
                "./node_modules/notosans-fontface/css/notosans-fontface-allweight.css",
                 // your app's global stylesheet
                "src/styles.scss"
            ]
```